cpp/tests/io/orc_test.cpp

```diff
@@ -1246,7 +1246,7 @@ TEST_F(OrcStatisticsTest, Overflow)
     not_too_small_seq, not_too_small_seq + num_rows, validity);
   table_view tbl({col1, col2, col3, col4});
 
-  auto filepath = temp_env->get_temp_filepath("OrcStatsMerge.orc");
+  auto filepath = temp_env->get_temp_filepath("OrcStatsOverflow.orc");
```

Comment:
What will happen with `OrcStatsMerge.orc`? And the tests associated with it?

Reply:
This is a temporary file created during this test. It can be named anything but the convention is to use a unique name per test, in case one wants to take a look at the file while debugging.

For this test, the file name seems to have been taken from `OrcStatisticsTest.Basic`